Specs/Core/BoundingSphereSpec.js

```diff
@@ -386,6 +388,23 @@ defineSuite([
         expect(sphere).toEqual(expected);
     });
 
+    it('fromOrientedBoundingBox', function() {
```

Comment:
We should have a test for both with and without a result parameter.  For example, see https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/Core/Matrix4Spec.js#L175